contributing.md

```diff
@@ -0,0 +1,46 @@
+# CONTRIBUTING
+
+ml-mining project is [Apache licensed](license.md) and accepts contributions via GitHub pull 
+requests. This document outlines some of the conventions on development workflow, commit message formatting, contact points, and other resources to make it easier to get your contribution accepted.
+
+## Certificate of Origin
+
+By contributing to this project you agree to the [Developer Certificate of Origin \(DCO\)](https://github.com/src-d/ml/tree/d1f13d079f57caa6338bb7eb8acb9062e011eda9/DCO/README.md). This document was created by the Linux Kernel community and is a simple statement that you, as a contributor, have the legal right to make the contribution.
+
+In order to show your agreement with the DCO you should include at the end of commit message, the following line: `Signed-off-by: John Doe <john.doe@example.com>`, using your real name.
+
+This can be done easily using the [`-s`](https://github.com/git/git/blob/b2c150d3aa82f6583b9aadfecc5f8fa1c74aca09/Documentation/git-commit.txt#L154-L161) flag on the `git commit`.
+
+## Support Channels
+
+The official support channels, for both users and contributors, are:
+
+* GitHub [issues](https://github.com/src-d/ml/issues)\*
+* Slack: \#machine-learning room in the [source{d} Slack](https://join.slack.com/t/sourced-community/shared_invite/enQtMjc4Njk5MzEyNzM2LTFjNzY4NjEwZGEwMzRiNTM4MzRlMzQ4MmIzZjkwZmZlM2NjODUxZmJjNDI1OTcxNDAyMmZlNmFjODZlNTg0YWM)
+
+\*Before opening a new issue or submitting a new pull request, it's helpful to search the project - it's likely that another user has already reported the issue you're facing, or it's a known issue that we're already aware of.
+
+## How to Contribute
+
+Pull Requests \(PRs\) are the main and exclusive way to contribute to the official sourced.ml
+.mining project. In order for a PR to be accepted it needs to pass a list of requirements:
```

Comment:
let's have the project name on the one line

Reply:
sorry, auto line break from pycharm. I just corrected it.